concourse/pipelines/create-bosh-cloudfoundry.yml

```diff
@@ -107,6 +107,13 @@ resources:
       region_name: {{aws_region}}
       versioned_file: bosh-CA.tar.gz
 
+  - name: logsearch-cert
```

Comment:
This commit message is wrong:

> Extract Terraform variables as certificate will use the system DNS zone name.

That was before, now you get the variable from the parameter, which is populated with the values from Makefile.

I would recommend avoid add in the comment what the code does in detail. Instead focus one explaining "why?", and overview of the "how" and maybe some caveats and links to references.